Khan/wonder-blocks
# Fix build config

## Summary:
When updating webapp to use wonder-blocks-data I ran into a very strange error. It was something along the lines of:

```
TypeError: Can't call n without new
```

I traced the issue back the to how we were building our packages.  'n' is actually 'KindError' from wonder-stuff-core.  We should probably be include dev bundles in our wonder-stuff-core to make debugging easier.

The issue with the wonder-blocks packages is that we were transpiling ES6 classes.  This was necessary due to a bunch of errors I Was seeing when we first switched over from @babel/preset-flow to @babel/preset-typescript.  In turns out that if we jettison @babel/plugin-transform-spread we can drop the class transforms as well.

~~In order to deal with the issue that @babel/plugin-transform-spread was addressing, I've remove the 'targets' option from @babel/env (this causes the preset to use our .browserslistrc file) and added core-js settings so the appropriate polyfills are added.  I've modified the rollup config to external all core-js modules.  We'll need to make sure that webapp is including those.  I've also updated the .browserslistrc file to reflect our supported browsers as documented in Confluence.~~

I've updated our babel config to match what's in webapp (minus the use of `@babel/plugin-transform-spread`).  That plugin will be remove from webapp's config when we migrate the static service to TypeScript.

Issue: None

## Test plan:
- `yarn build`
- look at packages/wonder-blocks-data/dist/index.js and the esm build
- see that classes aren't being compiled
- see that there are some core-js require (import) statements at the top of the file
